.github/auto_assign.yml

```diff
@@ -0,0 +1,2 @@
+addAssignees: author
```

Comment:
Does this need to be in a specific folder path? Otherwise I'd suggest to move it into a subfolder of sorts. Alternatively we could name it slightly differently to be more descriptive. For example as per the workflow: `platform.auto-assign-PR.configuration`. Just want to avoid ending up with an increasingly long list of files without any reference to why they are there or what they do.

A shame we need a file in the first place.

Reply:
Fixed. Let me know what you think.